database/count.sh

```diff
@@ -0,0 +1,17 @@
+source .env
+export PGPASSWORD=$DB_PASSWORD
```

Comment:
All of this without even having to enter a password, what a time to be alive

Reply:
I know right we are really living in the future now